src/converters.js

```diff
@@ -602,13 +602,15 @@ export function modelChangePostFixer( model, writer ) {
 				// In case of renamed element.
 				const item = entry.position.nodeAfter;
 
-				if ( item.hasAttribute( 'indent' ) ) {
+				// Only remove 'indent' attribute from items that doesn't allow such attribute. #103
```

Comment:
Could you add a comment that this code should be removed once https://github.com/ckeditor/ckeditor5-engine/issues/1228 is resolved?

Reply:
The whole check, am I righ?

Reply:
done